src/@ionic-native/plugins/camera-preview/index.ts

```diff
@@ -244,4 +252,14 @@ export class CameraPreview {
   })
   setFlashMode(flashMode?: string): Promise<any> { return; }
 
+  /**
+   * get supported picture size
+   * @return {Promise<any>}
+   */
+  @Cordova({
+    successIndex: 0,
```

Comment:
Thank you for your check.

Here are same issues. 
May I clean it at the same time?

```typescript
  /**
   * Stops the camera preview instance. (iOS & Android)
   * @return {Promise<any>}
   */
  @Cordova({
    successIndex: 0,
    errorIndex: 1
  })
  stopCamera(): Promise<any> { return; }

  /**
   * Switch from the rear camera and front camera, if available.
   * @return {Promise<any>}
   */
  @Cordova({
    successIndex: 0,
    errorIndex: 1
  })
  switchCamera(): Promise<any> { return; }

  /**
   * Hide the camera preview box.
   * @return {Promise<any>}
   */
  @Cordova({
    successIndex: 0,
    errorIndex: 1
  })
  hide(): Promise<any> { return; }

  /**
   * Show the camera preview box.
   * @return {Promise<any>}
   */
  @Cordova({
    successIndex: 0,
    errorIndex: 1
  })
  show(): Promise<any> { return; }
```

ref : https://github.com/cordova-plugin-camera-preview/cordova-plugin-camera-preview/blob/master/www/CameraPreview.js#L32-L46